app/javascript/packs/components/SimpleMap.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { Component } from "react"
+import { compose } from "recompose"
+import { withGoogleMap, GoogleMap, Marker, InfoWindow } from "react-google-maps"
+import FilterRow from "./FilterRow"
+
+const MapWithAMarker = compose(withGoogleMap)(props => {
+  console.log({ props })
+  return (
+    <GoogleMap
+      defaultZoom={12}
+      center={{
+        lat: props.center.latitude,
+        lng: props.center.longitude
+      }}
+    >
+      {props.markers.map(marker => {
+        const onClick = props.onClick.bind(this, marker)
+        return (
+          <Marker
+            key={marker.id}
+            onClick={onClick}
+            position={{ lat: marker.latitude, lng: marker.longitude }}
+          >
+            {props.selectedMarker === marker && (
+              <InfoWindow>
+                <div>{marker.shelter}</div>
+              </InfoWindow>
+            )}
+          </Marker>
+        )
+      })}
+    </GoogleMap>
+  )
+})
+
+export default class ShelterMap extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      shelters: [],
+      selectedMarker: false,
+      center: {
+        latitude: 29.7,
+        longitude: -95.3
```

Comment:
Can we set the default lat/lng from an env variable instead? Then each project can set their own map center.

Reply:
I think that’s a really good idea. Might need to put that in the json response tho because I don’t THINK we can out ruby ENV here.